README.md

```diff
@@ -1,32 +1,98 @@
-# Switch Statement
+# Switch Statements
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Explain what a switch statement is.
 
-## Introduction
+## What is a Switch Statement?
 
-A 1-2 sentence summary of what will be covered.
+A **switch statement** is a multiway branch that is more efficient than nested
+`if` statements in that it can have multiple execution paths. To help understand
+more about what goes into a switch statement, let us look at the syntax:
 
-## Topic 1
+```java
+switch (expression) {
+        // case statements
+        // values must be the same data type as the expression
+        case value1:
+        // statement sequence
+        break;    // breaks out of the case statement
+        
+    case value2:
+        // statement sequence
+        break;    // breaks out of the case statement
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+    // There can be multiple case statements
+    // and the default case statement is executed if none of the above
+    // cases are true
+    default:
+        // statement sequence
+        // no break is needed in the default case
+}
+```
 
-## Topic 2
+Duplicate cases are not allowed in switch statements.
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+The `break` statement is optional, although most programmers tend to use it to
```

Comment:
I think saying that the 'break' is optional might confuse students.
A forgotten 'break' is a very common cause of bugs. Perhaps you could supply a quick code sample to students to show what happens when a 'break' is not present.

Reply:
Good idea! I will also recommend using `break` statements over not.